dot/network/block_announce.go

```diff
@@ -208,9 +209,11 @@ func (s *Service) handleBlockAnnounceMessage(from peer.ID, msg NotificationsMess
 		return false, errors.New("invalid message")
 	}
 
-	if err = s.syncer.HandleBlockAnnounce(from, bam); err != nil {
-		return false, err
+	// TODO announce if we already have the block
```

Comment:
Is this addressed here? If not, is there an issue created for it?

Reply:
> Is this addressed here?

yes! removed the comment